solr/solr-ref-guide/src/monitoring-solr-with-prometheus-and-grafana.adoc

```diff
@@ -81,34 +81,44 @@ $ ./bin/solr-exporter -p 9854 -z localhost:2181/solr -f ./conf/solr-exporter-con
 
 === Command Line Parameters
 
-The parameters in the example start commands shown above:
+The list of available parameters for the Prometheus Exporter.
+All parameters can be provided via an environment variable, instead of through the command line.
 
 `h`, `--help`::
 Displays command line help and usage.
 
-`-p`, `--port`::
-The port where Prometheus will listen for new data. This port will be used to configure Prometheus. It can be any port not already in use on your server. The default is `9983`.
+`-p`, `--port`, `$PORT`::
+The port where Prometheus will listen for new data. This port will be used to configure Prometheus.
+It can be any port not already in use on your server. The default is `9983`.
```

Comment:
I find it puzzling that default port is the same as the default port for embedded Zookeeper, i.e. if you start `bin/solr -c` and then start exporter on same host, you'll have a port collision.

Reply:
That is very fair, and something we could certainly change in a major version, such as 9.0.

It should probably be done separately, since these changes are not backwards-incompatible.

Reply:
https://issues.apache.org/jira/browse/SOLR-14972